src/components/app/details/cdDetails/service.ts

```diff
@@ -77,7 +77,7 @@ const prepareDeploymentTemplateData = (rawData): Record<string, DeploymentHistor
     return deploymentTemplateData
 }
 
-const preparePipelineConfigData = (rawData): Record<string, DeploymentHistorySingleValue> => {
+export const preparePipelineConfigData = (rawData): Record<string, DeploymentHistorySingleValue> => {
     const pipelineConfigData = {}
     if (rawData['pipelineTriggerType']) {
```

Comment:
Same for `pipelineTriggerType`

Reply:
Same here. We're just accessing the `pipelineTriggerType` value from rawData.